client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -179,7 +182,7 @@ class BusinessDetails extends Component {
 				{ extensionBenefits.map( benefit => (
 					<div className="woocommerce-profile-wizard__benefit" key={ benefit.title }>
 						<div className="woocommerce-profile-wizard__business-extension">
-							<img src={ wcSettings.wcAdminAssetUrl + benefit.icon } alt="" />
+							<img src={ wcAdminAssetUrl + benefit.icon } alt="" />
```

Comment:
img->svg

Reply:
As mentioned elsewhere, I think this should be addressed in it's own pull.

Reply:
https://github.com/woocommerce/woocommerce-admin/issues/2943